pkg/client/client.go

```diff
@@ -24,10 +24,10 @@ import (
 var logger = log.Logger("keep-ecdsa")
 
 const (
-	awaitingKeyGenerationLookback = 24 * time.Hour
-	keyGenerationTimeout          = 120 * time.Minute
-	signingTimeout                = 90 * time.Minute
-	blockConfirmations            = 12
+	defaultAwaitingKeyGenerationLookback = 24 * time.Hour
+	defaultKeyGenerationTimeout          = 3 * time.Hour
+	defaultSigningTimeout                = 2 * time.Hour
+	blockConfirmations                   = 12
```

Comment:
```
// The default number of block confirmations the client waits for until
// it starts the requested signing process. This value prevents from
// reporting unauthorized signings by adversaries in case of a chain
// fork.
```